src/pages/Login.vue

```diff
@@ -67,6 +68,36 @@ export default {
         }
       )
     },
+    loginFB() {
```

Comment:
1)why need to seperate into 2 functions: loginCustumerFB() , loginFB()
can put "window.FB.login(res => { ... into vuex action loginFB ?
2)same as above for Register
3)wrong Custumer -> Customer , FB -> Fb